src/app/feed/feed-create-post/feed-create-post.component.ts

```diff
@@ -389,20 +379,13 @@ export class FeedCreatePostComponent implements OnInit {
     const action = this.postToEdit ? "edit" : "create";
     this.backendApi
       .SubmitPost(
-        this.globalVars.localNode,
         this.globalVars.loggedInUser?.PublicKeyBase58Check,
         post.editPostHashHex /*PostHashHexToModify*/,
         this.isReply ? this.parentPost?.PostHashHex ?? "" : parentPost?.PostHashHex ?? "" /*ParentPostHashHex*/,
-        "" /*Title*/,
         bodyObj /*BodyObj*/,
         repostedPostHashHex,
         postExtraData,
-        "" /*Sub*/,
-        // TODO: Should we have different values for creator basis points and stake multiple?
-        // TODO: Also, it may not be reasonable to allow stake multiple to be set in the FE.
-        false /*IsHidden*/,
-        this.globalVars.defaultFeeRateNanosPerKB /*MinFeeRateNanosPerKB*/,
-        this.inTutorial
+        false /*IsHidden*/
```

Comment:
I saw some changes for passing `isHidden` param in different place, just want to make sure that it should be changed here

Reply:
I made a hide-reversion button for blogs, but it's not relevant here for regular posts. Good eye though!